### ISSUE TITLE
Get FlyleafSharedOverlay DetachedContent as Visual

### ISSUE THREAD

**Issue by Agostino44** [SuRGeoNix/Flyleaf#459]
I have a UserControl with a FlyleafSharedOverlay that has a Canvas as content that contains various elements and a DetachedContent that contains one or more Flyleafhosts for video playing on background.
`<UserControl >
    <fl:FlyleafSharedOverlay Name="main_host">
        <fl:FlyleafSharedOverlay.DetachedContent>
                <Canvas x:Name="detached_canvas" Background="Black" AllowDrop="True">
                    
                </Canvas>
        </fl:FlyleafSharedOverlay.DetachedContent>
          <Canvas Name="MosaicCanvas" Background="Transparent" Panel.ZIndex="0">
              
          </Canvas>
    </fl:FlyleafSharedOverlay>
</UserControl>`
This Usercontrol is in a window that is always open and I view playing different effects and videos.
So far, so good.
Sometimes I need to open another window that displays exactly what is currently running in the first main window. A sort of mirror of the first window.
I thought of structuring the mirror window in this way:
`<Window>
        <Rectangle x:Name="mirror_grid" HorizontalAlignment="Stretch" VerticalAlignment="Stretch">
            <Rectangle.Fill>
                <VisualBrush x:Name="mirror_visual" Stretch="Fill" Visual="{Binding}" TileMode="None"></VisualBrush>
            </Rectangle.Fill>
        </Rectangle>
</Window>`
than I call and show this window assigning to the window DataContext the UserControl with the FlyleafSharedOverlay. By doing this I don't see anything in the mirror window. The only way I've found to see something is by assigning to DataContext the property Overlay of the FlyleafSharedOverlay, but this way I only see what is in the canvas MosaicCanvas, I'm not able to see the content of FlyleafSharedOverlay.DetachedContent in the mirror window.
This probably happens because the DetachedContent is in the Surface Window that is separated from Overlay window.
Is there a way to see in the visualbrush also the DetachedContent?
Thank you.

**Comment by SuRGeoNix** [comment]
@Agostino44 Can you create a simplified demo so I can see what you are trying to do? I'm not sure if you want to mirror or just transfer the content. Also I'm not sure if you want to mirror the players (surfaces) the overlay content or both. Did you try to use the replica players instead as it would be better for performance if you want to mirror the players/surfaces?

**Comment by SuRGeoNix** [comment]
@Agostino44 Indeed, Wiki and Samples need more work. You can find an interactive zoom and replica player for WPF under Tests/WpfFlyleafHost and in App.xaml use "OverlayStandAlone.xaml" as StartupUri. It sets the whole window as Surface with this:-

```csharp
 FlyleafHost = new FlyleafHost(this) { ... Player = Player }
```
and passes the same Player to the replica FlyleafHost with this xaml:-

```xaml
<fl:FlyleafHost x:Name="FlyleafHostReplica" ReplicaPlayer="{Binding Player}" ...
```
So generally, to replicate another player within a FlyleafHost instead of setting the Player you need to set the ReplicaPlayer.

**Comment by Agostino44** [comment]
I can't set the zoom on the ReplicaPlayer.

`player_host.ReplicaPlayer.SetZoom(zoom);`

it has no effect.
any suggestions on how to apply a zoom on the replicaplayer?

Thanks.

**Comment by SuRGeoNix** [comment]
@Agostino44 Let me know if that works for you

**Comment by SuRGeoNix** [comment]
@Agostino44 sorry for the delay, nuget is out
